### DISCUSSION TITLE
Howto upgrade fossology from 4.1.0 to 4.4.0

### DISCUSSION THREAD

**Discussion by petermezei79** [fossology/fossology#2831] [Q&A]
Hi, 
I'm using the docker version of fossology, and i want to upgrade from 4.1.0 to 4.4.0. I tried to spin up a new 4.4.0 docker container with the same config as my old 4.1.0, but this container can't start, i got this error:
`*** update the database and license_ref table ***
Old release was unknown
PHP Warning:  pg_query(): Query failed: ERROR:  cannot drop table agent column agent_rev_old because other objects depend on it
DETAIL:  view runstat_up_agent depends on table agent column agent_rev_old
HINT:  Use DROP ... CASCADE to drop the dependent objects too. in /usr/local/share/fossology/lib/php/Db/Driver/Postgres.php on line 68
/usr/local/share/fossology/lib/php/libschema.php.CRITICAL: ERROR:  cannot drop table agent column agent_rev_old because other objects depend on it DETAIL:  view runstat_up_agent depends on table agent column agent_rev_old HINT:  Use DROP ... CASCADE to drop the dependent objects too. [] []
PHP Fatal error:  Uncaught Fossology\Lib\Exception: error executing: ALTER TABLE "agent" DROP COLUMN "agent_rev_old"

ERROR:  cannot drop table agent column agent_rev_old because other objects depend on it
DETAIL:  view runstat_up_agent depends on table agent column agent_rev_old
`
I'm using an external postgres database.
What sould i do? Thanks!

**Comment by shaheemazmalmmd** [comment]
Please refer to comment from @GMishx here 

https://github.com/fossology/fossology/issues/2679#issuecomment-2208929259
